### ISSUE TITLE
WFS GetFeature via POST

### ISSUE THREAD

**Issue by Roel** [geopython/OWSLib#439]
Is there support for sending WFS GetFeature requests via POST? I'd like to query the WFS service using a prepared WFS GetFeature filter in XML, but I can't get it to work with the current version.

The 'method' parameter seems to suggest it should work:
https://github.com/geopython/OWSLib/blob/d2a9722685de09e4f70189cdc034a23f5ecccffa/owslib/feature/wfs110.py#L154-L157
But changing this to `method='Post'` and providing a filter as XML doesn't seem to work.

I think the culprit is in the
https://github.com/geopython/OWSLib/blob/d2a9722685de09e4f70189cdc034a23f5ecccffa/owslib/feature/wfs110.py#L247

I can prepare a PR to support a (fourth) way of using the GetFeature method, with just a filter (in XML) and method = 'Post'? Like so:
```
wfs = WebFeatureService(
                url="https://www.dov.vlaanderen.be/geoserver/wfs",
                version="1.1.0")

filter = """<wfs:GetFeature service="WFS" version="1.1.0"
  xmlns:wfs="http://www.opengis.net/wfs"
  xmlns:ogc="http://www.opengis.net/ogc"
  xmlns:gml="http://www.opengis.net/gml"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://www.opengis.net/wfs
                      http://schemas.opengis.net/wfs/1.1.0/wfs.xsd">
  <wfs:Query typeName="gw_meetnetten:meetnetten">
    <ogc:Filter>
      <ogc:And>
      <ogc:Within>
        <ogc:PropertyName>geom</ogc:PropertyName>
        <gml:Polygon srsName="http://www.opengis.net/gml/srs/epsg.xml#31370"><gml:exterior><gml:LinearRing><gml:posList>145507.928000719 216606.507469683 150613.092722679 214309.409048491 152771.263602172 213891.622205293 151333.251448721 211139.985945256 153660.550282115 211009.519252685 154511.34476093 212074.93472839 153622.622518193 213748.657664443 153597.457095669 215434.484793785 152901.365702482 217023.736349089 151546.817465871 218803.502095865 150001.618076775 220188.887311176 148313.364709693 220881.823122265 146217.66190153 220381.325132092 144289.396255128 220048.65719942 143581.546259324 218986.368908396 145592.876194333 217873.164540651 145507.928000719 216606.507469683</gml:posList></gml:LinearRing></gml:exterior></gml:Polygon>
      </ogc:Within>
      <ogc:PropertyIsEqualTo>
      <ogc:PropertyName>gemeente</ogc:PropertyName>
      <ogc:Literal>Mechelen</ogc:Literal>
      </ogc:PropertyIsEqualTo>
      </ogc:And>
    </ogc:Filter>
  </wfs:Query>
</wfs:GetFeature>"""

result = wfs.getfeature(filter=filter, method='Post')
```

Are we in favor of supporting such a use-case?

**Comment by huard** [comment]
We're hitting size limits on our GET feature requests, so having support for POST would be great.

**Comment by gilav** [comment]
Hello

Post is not working for me. Is this issue solved in current branch?

**Comment by huard** [comment]
This can be closed now with #706.
